aksel.nav.no/website/app/dev/produktbloggen/layout.tsx

```diff
@@ -0,0 +1,17 @@
+import Footer from "@/app/_ui/footer/Footer";
+import { Header } from "@/app/_ui/header/Header";
+import styles from "./_ui/Produktbloggen.module.css";
+
+export default async function DesignsystemLayout({
+  children,
+}: {
+  children: React.ReactNode;
+}) {
+  return (
+    <div className={styles.websitePage}>
+      <Header className={styles.header} />
+      <div className={styles.pageLayout}>{children}</div>
+      <Footer />
+    </div>
```

Comment:
```suggestion
    <Page className={styles.websitePage} footer={<Footer />}>
      <Header className={styles.header} />
      <div className={styles.pageLayout}>{children}</div>
    </Page>
```

Can prob remove some CSS from `styles.pageLayout` since pages handles some here.

Could the `div` here also be `main` so that each page don't need to set it itself?
```
   <Page className={styles.websitePage} footer={<Footer />}>
      <Header className={styles.header} />
      <div className={styles.pageLayout}>{children}</div>
    </Page>
```